server.go

```diff
@@ -126,8 +126,16 @@ func runMain() error {
 
 	if err != nil {
 		application.Close()
-		return fmt.Errorf("Failed to initialize the main controller: %v",
-			err)
+		return fmt.Errorf("Failed to initialize the main controller: %v", err)
+	}
+
+	// Check that dcrstakepool config and all stakepoold configs
+	// have the same value set for `coldwalletextpub`.
+	if !controller.Cfg.StakepooldServers.ColdWalletExtPubMatches(cfg.ColdWalletExtPub) {
+		err := fmt.Errorf("coldwalletextpub is not the same in " +
```

Comment:
You don't have to log here, it will be logged after returned.

Currently:
![Untitled](https://user-images.githubusercontent.com/34998433/64906632-cb879b80-d723-11e9-8d7a-30907118e1fd.png)

Also, we have only been wrapping comments to 80 columns length, because it may go into godoc, I believe is the reason.  You don't have to wrap here, but I really don't mind if it's easier for you to read.

Reply:
Yeah, I find shorter-length comments easier to read.